package.json

```diff
@@ -17,12 +17,12 @@
   },
   "scripts": {
     "test": "xo",
-    "build": "cd swift && xcodebuild archive -derivedDataPath $(mktemp -d) -scheme aperture",
+    "build": "cd swift && xcodebuild archive -derivedDataPath $(mktemp -d) -scheme aperture && mv main ../aperture",
```

Comment:
Can you change this in the Xcode "Run Script" instead? That way we can use Xcode to build too. Good when developing.